src/Base/UnitsSchemaInternal.cpp

```diff
@@ -360,6 +360,24 @@ QString UnitsSchemaInternal::schemaTranslate(const Quantity &quant, double &fact
             factor = 1e12;
         }
     }
+    else if (unit == Unit::ElectricalConductivity) {

```

Comment:
These factors are all wrong. When you have S/m then you must express this with the internal units where length is measured in mm.
1 S/m = A² * s³ / (kg * m³) = A² * s³ / (kg * (1000 mm)³) = 10⁻⁹ A² * s³ / (kg * mm³)

So the factors are:
mS/m -> 1e-12
S/m -> 1e-9
kS/m -> 1e-6
MS/m -> 1e-3

Reply:
> These factors are all wrong. 
> 1 S/m = A² * s³ / (kg * m³)

Thanks! How embarrassing for me. I only cared about the denominator despite Siemens itself is a combined unit.